src/components/LanguageSelection/mocks.js

```diff
@@ -0,0 +1,77 @@
+import {rest} from 'msw';
+
+export const BASE_URL = 'http://localhost:8000/api/v2/';
```

Comment:
gradual improvement/refactoring - I imagine this will at some point live in `src/mocks` or something like that.